lib/app/modules/us_core_r4/us_core_allergyintolerance_sequence.rb

```diff
@@ -161,16 +162,18 @@ def validate_resource_item(resource, property, value)
           'AllergyIntolerance.patient'
         ]
         must_support_elements.each do |path|
-          truncated_path = path.gsub('AllergyIntolerance.', '')
-          already_found = @instance.must_support_confirmed.include?(path)
-          element_found = already_found || can_resolve_path(@allergyintolerance, truncated_path)
-          skip "Could not find #{path} in the provided resource" unless element_found
-          @instance.must_support_confirmed += "#{path}," unless already_found
+          @allergyintolerance_ary.each do |resource|
+            truncated_path = path.gsub('AllergyIntolerance.', '')
+            already_found = @instance.must_support_confirmed.include?(path)
+            element_found = already_found || can_resolve_path(resource, truncated_path)
+            skip "Could not find #{path} in the provided resource" unless element_found
```

Comment:
Skip will cause the whole test to skip on the first resource, if the first resource doesn't have all the elements, right?  The idea is that it should skip if by the end of looking at every resource I never saw at least one element.

Reply:
should be fixed now